src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -27,7 +27,7 @@
     "require-dev": {
         "symfony/asset": "^4.4|^5.0",
         "symfony/stopwatch": "^4.4|^5.0",
-        "symfony/dependency-injection": "^4.4|^5.0",
+        "symfony/dependency-injection": "^5.2",
```

Comment:
is the bundle really usable without `symfony/dependency-injection` or should this become a requirement rather than `require-dev` + `conflict` ?

Reply:
No need to break BC here IMHO (in any case, not related to this PR).

Reply:
I don't think adding it as an actual requirement would be a BC break. But I agree it is not really related to this PR